tutorials/inputs/input_examples.rst

```diff
@@ -203,14 +203,14 @@ the :kbd:`T`:
     {
         if (inputEvent is InputEventKey keyEvent && keyEvent.Pressed)
         {
-            if ((KeyList)keyEvent.Scancode == KeyList.T)
+            if ((Keylist)keyEvent.Keycode == KeyList.T)
```

Comment:
This is wrong, `KeyList` is the correct casing.

Reply:
Opened #4566 to fix it, thanks for the heads up